keyboards/aki27/cocot46plus/keyboard.json

```diff
@@ -0,0 +1,133 @@
+{
+  "keyboard_name": "cocot46plus",
+  "manufacturer": "aki27",
+  "maintainer": "markstos",
+  "tags": ["unibody","ergo","corne","trackball","encoder","rgb"]
+  "usb": {
+    "device_version": "1.0.0",
+    "pid": "0x0003",
+    "vid": "0x1727"
+  },
+  "development_board": "promicro",
+  "build": {
+    "lto": true
+  },
+  "features": {
+    "bootmagic": true,
+    "encoder":true,
+    "extrakey": true,
+    "mousekey": true,
+    "oled": true,
+    "pointing_device": true,
+    "rgblight": true
+  },
+  "matrix_pins": {
+    "cols":  ["F4", "F5", "F6", "F7", "B1", "B3"],
+    "rows": ["D4", "C6", "D7", "E6", "NO_PIN"],
+    "custom": true,
+    "direct": [
+    [ "A00",   "A01",   "A02",   "A03",   "A04",   "A05" ], 
+    [ "A10",   "A11",   "A12",   "A13",   "A14",   "A15" ], 
+    [ "A20",   "A21",   "A22",   "A23",   "A24",   "A25" ], 
+    [ "A30",   "A31",   "A32",   "A33",   "A34",   "A35" ], 
+    [ null,    null,    "A42",   null,   null,    "A45" ], 
+    [ "B00",   "B01",   "B02",   "B03",   "B04",   "B05" ], 
+    [ "B10",   "B11",   "B12",   "B13",   "B14",   "B15" ], 
+    [ "B20",   "B21",   "B22",   "B23",   "B24",   "B25" ], 
+    [ "B30",   "B31",   "B32",   "B33",   "B34",   "B35" ], 
+    [ null,    null,    "B42", null,    null,     "B45" ]  
+    ]
+  },
```

Comment:
I've tested the keyboard with built with this firmware and it works. What's not correct here?

Reply:
row-col matrix and direct pins cannot be used simultaneously. The values of this direct pins array do not make any sense anyway. Additionally, since you have a custom `matrix.c` the col and row pin arrays should not be specified in keyboard.json.

Reply:
Thanks. Perhaps the keyboard.json validator could throw an error of `cols`/`rows` and `direct` are both  used. Sounds like I can remove them both.